storybook/stories/assets/fake_api.json

```diff
@@ -0,0 +1,202 @@
+[
```

Comment:
@erikyo do we need a full copy of this file, or can we adapt the changes to the one already in `includes/restapi/fake_api.json`?

Reply:
I copied the original fake_api file (well, was moved before the merge) because in the future will be removed from `/includes/`. For the moment we can link to the "oringinal" and remove this one, but then we will have to remember to move it again when we have a working api

Reply:
@erikyo Do you just want to move it now? Into something like a `data` folder in the project root. That way it has a permanent home that both can use, and makes sense when the PHP code no longer requires it.

Reply:
yep I want to move it, but not in this PR! For the moment I did as you suggested, deleted the copy and linked it to the "original" one.